lib/main.dart

```diff
@@ -116,7 +116,10 @@ void main(List<String> args) async {
     }
 
     // Report success; print version.
-    await promiseToFuture(exec.exec('dart', ['--version']));
+    await promiseToFuture(exec.exec(
+      'dart',
+      ['--version'].map((str) => str.toJS).toList().toJS,
```

Comment:
Oh, forgot to mention - I think you can probably just do `['--version'.toJS].toJS`.

Reply:
👍  - simplified